src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -1685,5 +1685,28 @@ sap.ui.define([
 		return this.bSecureSearch;
 	};
 
+	/**
+	 * Determines how the next link url should be used
+	 *
+	 * @param {string} sNextLinkUrl The next link url
+	 * @param {string} sPath The FHIR resource path
+	 * @param {object} mParameters Existing parameters
```

Comment:
This `mParameters` object could described in a separate `@type`. Its not clear which parameters can/should be there

Reply:
done